convertSynapse2Graph.py

```diff
@@ -48,47 +49,52 @@ def idGenerator(start=0):
 
 def getEntities(projectId):
```

Comment:
merge with `getEntities` that is in `activities2Graph.py` (or move to a shared library file and call the same one)